### PR TITLE
fix: show filler image on failure to load avatar in about us contributor

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] (if relevant) I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] (if relevant) I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3943** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3943" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3943?utm_source=stack-comment).

File: src/components/AboutDashboard/components/BuiltByTheBestSection.tsx [hunk 1]
```diff
@@ -99,13 +99,20 @@ const ContributorCard: React.FC<
     avatarUrl: string | undefined
   }>
 > = ({ name, title, avatarUrl }) => {
+  const [validAvatar, setValidImage] = useState(true)
```
Comment (peripheralist) [inline src/components/AboutDashboard/components/BuiltByTheBestSection.tsx:102]: `validImage` or `validAvatar` 🤔

lol approving

**Review by peripheralist — APPROVED** [review]

File: src/components/AboutDashboard/components/BuiltByTheBestSection.tsx (hunk 1 above)
Comment (wraeth-eth) [reply src/components/AboutDashboard/components/BuiltByTheBestSection.tsx:102]: my bad - will fix. identity crisis lol

**Comment by wraeth-eth** [comment]
@wraeth-eth merged this pull request with [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3943).

**Pull request merged**